static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js

```diff
@@ -20,22 +30,52 @@ export const acquisitionsBannerFivTemplate = (
     </div>
     <div class="fiv-banner__container">
         <div class="fiv-banner__text">
-            This is a test banner for FIV - if you see it please email userhelp@theguardian.com
+        ${
+            window.location.hash.match(/[#&]fiv-potential(&.*)?$/) ||
+            (shouldSeeReaderRevenue &&
+                !window.location.hash.match(/[#&]fiv-existing(&.*)?$/))
+                ? `<div class="fiv-banner__headline fiv-banner__wide">
+                ${potentialTitleWide.markup}
+            </div><div class="fiv-banner__headline fiv-banner__mobile">
+                ${potentialTitleMobile.markup}
+            </div>
+            <div class="fiv-banner__text_bold">
+                …we remain editorially independent, our journalism free from commercial bias and our reporting open and accessible to all.
+            </div>
+            <div class="fiv-banner__text_normal">
+                Imagine what we could continue to achieve with the support of many more of you. Together we can be a force for change.
+            </div>`
+                : `<div class="fiv-banner__headline">
```

Comment:
this is hard to read with current formatting

Reply:
`make fix` is not helpful in this case...